### PR TITLE
hw01-Jingkun Wang 

### PR DESCRIPTION
**Pull request by jingkunw**
This is my first time I used Github and R. Many new terms, like"Markdown", "pull request", "commit", made me confused. I googled these terms to try to understand them better. As for the actual workflow of this homework, I used #, ##, ### to distinguish the different levels of headings. There is something need to pay attention. We need to enter space before the #, if not, it will not be shown in the form that we want.  In terms of the "lists", I did the unordered lists first, but it didn't work well. So, I used ordered one here. The code of the image was copied from the Github browser, which was generated after I dragged the image into the "write" area under the function of creating a new issue. I got this instruction for Youtube. Beside Youtube and Google, the guide of Markdown also helped a lot. I learned how to insert links just from there.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,32 @@
...
+
+1. Watching TV 
+1. Traveling
+1. Playing Guzheng
+1. Eating
+
+
+##This is me
+
+<img src= "https://user-images.githubusercontent.com/32334821/31059931-0992ce40-a6cf-11e7-840e-d42f6f18ed0c.jpg" width="200" height="400"/>
```
Comment (AymanBaig) [inline README.md]: This image actually did not appear here.

**Review by AymanBaig — COMMENTED** [review]
Evaluation HW-01 for Jingkun Wang
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |           |      x        |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |           |    x          |            |
| **Ease of access**          |  x         |              |            |

Remarks:
* Elaborate on above, especially for "needs work." I was not able to follow the link or view the image. 
* Some specific praise? Jingkun was able to do most of the required coding. Multiple outside sources were also used in order to do the homework which will probably come in handy throughout the course. 
* Something I learned? I learned that YouTube tutorials will be helpful in terms of coding. I will surely use them later on.
* Specific constructive criticism? Another student added an image using this code: +. This can help you add the image appropriately.
* Something I know and that you, my peer, might like to know because it is relevant to something you struggled with. I was able to use the R Markdown website with help on these basic functions. That would also be a helpful resource.

**Comment by sushmitavgopalan16** [comment]
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |        x   |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |      x     |              |            |

Remarks:

* Nice work! You've shown proficient use of various different elements.
* Nice use of headers.
* You can remove the text that was in the file already - "hw01" etc.
* Commit early and often for future, longer assignments - helps you keep track and gives you versions to go back to, if required!

**Comment by zahraamalias** [comment]
Evaluation HW-01 for Jingkun Wang
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            | v          |              |            |
| **Coding strategy**         |v           |              |            |
| **Presentation**            |           |v              |            |
| **Achievement, creativity** |           |v              |            |
| **Ease of access**          |v           |              |            |

Remarks:

* Although you have attempted to use "#", "##", or "###", when put together next to each other they don't look that easy to distinguish and the page seemed 'full'. Try to alternate normal text with these headers for better presentation.
* While you have used bullet points well, you could also italicize or bold the text to improve presentation.
* It's good that the image showed up and the link worked. Well done!

**Pull request closed without merging**